docker/Dockerfile

```diff
@@ -62,7 +62,7 @@ RUN set -x \
   && apt-get install -y --no-install-recommends $buildDeps \
   && pip install -r /tmp/requirements-frozen.txt \
   && mkdir /tmp/uwsgi-dogstatsd \
-  && wget -O - https://github.com/eventbrite/uwsgi-dogstatsd/archive/filters-and-tags.tar.gz | \
+  && wget -O - https://github.com/DataDog/uwsgi-dogstatsd/archive/1a04f784491ab0270b4e94feb94686b65d8d2db1.tar.gz | \ # pinned the same as in getsentry
```

Comment:
```suggestion
  # pinned the same as in getsentry
  && wget -O - https://github.com/DataDog/uwsgi-dogstatsd/archive/1a04f784491ab0270b4e94feb94686b65d8d2db1.tar.gz | \
```

Reply:
Yup, c37f59f90c48b1ee2f1445b34f1ba314bb4e40dc.

Reply:
Trying out the comment in 15b2ba224696abc74857045582ed7a3bb08bb6aa, hope it parses right.

Reply:
it will, but that looks weirder than mine -- docker discards lines which are just comments

Reply:
> docker discards lines which are just comments

Dope. 👍

> that looks weirder than mine

And more contextually relevant, since it is aligned with the pinned SHA. ☺️